sys/include/net/coap.h

```diff
@@ -49,6 +49,7 @@ extern "C" {
 #define COAP_OPT_BLOCK1         (27)
 #define COAP_OPT_PROXY_URI      (35)
 #define COAP_OPT_PROXY_SCHEME   (39)
+#define COAP_OPT_NO_RESPONSE    (258)   /**< suppress CoAP response (RFC 7968) */
```

Comment:
Bordering on nit-picky and potentially more noisy than necessary, but

```suggestion
/**
 * @brief suppress CoAP response
 * @see [RFC 7968](https://datatracker.ietf.org/doc/html/rfc7967)
 */
#define COAP_OPT_NO_RESPONSE    (258)
```

?